finclab/my-wealth-frontend
# Title: implement add bvn details feature

#### What does this PR do?
- This PR implements the add and link User BVN details to the user account.

#### Description of Task to be completed?
- Implement a functionality that consumes the add bvn details end point.
- Implement test for add bvn details component.
- Implement test for add bvn details actions.
- Implement test for add bvn reducers.

#### How should this be manually tested?
- Clone the repo `<github url>`
- Or do a `git pull --rebase origin <branch name>`
- Cd into the project directory and run `npm install` to install all dependencies.
- Visit `http://localhost:3000` on browser and login as an authenticated user
- Click on verify me on the side navigation then click on verify bvn.
- Input the user bvn (for testing input `11111111111`) and get redirected to the add bvn details page.
- Click on continue to add user bvn.

#### Any background context you want to provide?
<img width="684" alt="Screen Shot 2019-05-03 at 8 27 56 AM" src="https://user-images.githubusercontent.com/32917511/57124357-68512a00-6d7d-11e9-99eb-496c42645a54.png">

#### What are the relevant pivotal tracker stories?(if applicable)
[#165785023](https://www.pivotaltracker.com/story/show/165785023)
